### PR TITLE
Update Exchange-Teams-interact.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5368.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by dariomws** [comment]
@JohanFreelancer9 give me some hours, I need to continue working on this PR.

**Comment by dariomws** [comment]
@JohanFreelancer9 you are good to go here, thanks for waiting!

@dstrome could you kindly review these updates?

I want to put this article in a discussion as I think the article was inconsistent because it said in two different places that you need an Exchange hybrid, but really for almost all features with AD Sync and OAuth is enough in the Exch on-prem scenario. For calendar, you also need a mailbox located in Exch 2016 CU3+ and Autodiscover/EWS.
Do we need Exch 2016 CU3+ and Autodiscover/EWS to modify the user profile picture?

@carolynblanding maybe you kindly give me your point of view.

@cichur "Requirements with on-premises Exchange mailbox user" section is not clear for me. I think it's almost the same we see above. Should we leave it there? Is Teams' calendaring with on-prem mailbox supported on Exch 2013 CU19+ also?

Source: https://techcommunity.microsoft.com/t5/exchange-team-blog/configuring-teams-calendar-access-for-exchange-on-premises/ba-p/1484009

Thanks to everyone!

**Comment by cichur** [comment]
Hi @darionws, 
The support engineers requested that section be added to the article. I believe they wanted to point out some common issues that received many support calls. I can run your questions by them to see if they have concerns, but they did review the article before I published those changes.

**Comment by dariomws** [comment]
> Hi @dariomws
> The support engineers requested that section be added to the article. I believe they wanted to point out some common issues that received many support calls. I can run your questions by them to see if they have concerns, but they did review the article before I published those changes.

Thanks for your reply @cichur. I think it's a good idea to ask them again because it seems to be redundant information. Also, I think Exchange 2013 is not supported for Teams' calendar. All this information is also detailed in the blog article.

**Comment by carolynblanding** [comment]
The endless Exchange/Teams interaction.

I put this together internally based on deep dive and testing. It’s my source of truth.
https://dev.azure.com/Supportability/UC/_wiki/wikis/UC.wiki/9093/How-Teams-and-Exchange-Interact

Depending on workloads, different bits are required and I broke the article down to what is required starting with minimum.





Carolyn Blanding

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -25,29 +25,26 @@ appliesto:
...
+Users with mailboxes hosted on-premises must be synchronized to Azure Active Directory, and OAuth authentication should be configured as described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://docs.microsoft.com/exchange/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help) and working between you Exchange Online and Exchange on-premises (We highly recommended to run Hybrid Configuration Wizard to configure full hybrid mode). They can make use of almost all the features in the Exchange Online or vNext scenario with some considerations. To enable calendar delegation for these users, you must also complete steps 2-3 as described in [Configure Integration and OAuth between Skype for Business Online and Exchange Server](https://docs.microsoft.com/skypeforbusiness/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises); these steps will provide the Teams scheduling application the required permissions to confirm delegate permissions.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:34]:
```suggestion
Users with mailboxes hosted on-premises must be synchronized to Azure Active Directory, and OAuth authentication should be configured as described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://docs.microsoft.com/exchange/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help) and working between you Exchange Online and Exchange on-premises (we highly recommend running Hybrid Configuration Wizard to configure full hybrid mode). They can make use of almost all the features in the Exchange Online or vNext scenario with some considerations. To enable calendar delegation for these users, you must also complete steps 2 and 3 as described in [Configure Integration and OAuth between Skype for Business Online and Exchange Server](https://docs.microsoft.com/skypeforbusiness/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises); these steps will provide the Teams scheduling application with the required permissions to confirm delegate permissions.
```

Comment (JohanFreelancer9) [reply Teams/Exchange-Teams-interact.md:34]: Please change:
1. (**we** highly **recommend** **running** Hybrid Configuration Wizard to configure full hybrid mode)

File: Teams/Exchange-Teams-interact.md [hunk 2]
```diff
@@ -73,7 +70,7 @@ Microsoft Teams works with several Microsoft 365 and Office 365 services to prov
...
+- To let Microsoft Teams work with Exchange on-premises, you must configure the new Exchange OAuth authentication protocol, preferably by running the Exchange Hybrid Wizard, as described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://docs.microsoft.com/exchange/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help). To make calendar access work for your on-prem mailboxes, Teams needs access to your Exchange on-prem organization for both Autodiscover and EWS, and mailbox should be located on Exchange 2016 CU3 or later. To enable users with Exchange on-premises mailbox to schedule Teams meetings on behalf of another user, you must must also complete steps 2-3 as described in [Configure Integration and OAuth between Skype for Business Online and Exchange Server](https://docs.microsoft.com/skypeforbusiness/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises).
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:73]:
```suggestion
- To let Microsoft Teams work with Exchange on-premises, you must configure the new Exchange OAuth authentication protocol, preferably by running the Exchange Hybrid Wizard, as described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://docs.microsoft.com/exchange/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help). To make calendar access work for your on-premises mailboxes, Teams needs access to your Exchange on-premises organization for both Autodiscover and EWS, and mailboxes should be located on Exchange 2016 CU3 or later. To enable users with Exchange on-premises mailboxes to schedule Teams meetings on behalf of another user, you must also complete steps 2 and 3 as described in [Configure Integration and OAuth between Skype for Business Online and Exchange Server](https://docs.microsoft.com/skypeforbusiness/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises).
```

Comment (JohanFreelancer9) [reply Teams/Exchange-Teams-interact.md:73]: Suggested changes:
To make calendar access work for your **on-premises** mailboxes, Teams needs access to your Exchange **on-premises** organization for both Autodiscover and EWS, and **mailboxes** should be located on Exchange 2016 CU3 or later. To enable users with Exchange on-premises mailboxes to schedule Teams meetings on behalf of another user, you **must** **(please remove double word)** also complete steps **2 and 3** as described in...

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cichur** [comment]
Hi, I'm waiting to merge the pull request until we get some clarification from the support team. I've started a conversation with them. Cindy

**Pull request merged**